File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples22.vb
' <snippetlinqexamples22>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_gele1 = From c In svcContext.ContactSet _
                    Where c.CreditLimit.Value >= 200 _
                    AndAlso c.CreditLimit.Value <= 400 _
                    Select New With {Key c.FirstName,
                                     Key c.LastName}
  For Each c In query_gele1
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName)
  Next c
 End Using

' </snippetlinqexamples22>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples22.cs
// <snippetlinqexamples22>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_gele1 = from c in svcContext.ContactSet
                    where c.CreditLimit.Value >= 200 &&
                    c.CreditLimit.Value <= 400
                    select new
                    {
                     c.FirstName,
                     c.LastName
                    };
  foreach (var c in query_gele1)
  {
   System.Console.WriteLine(c.FirstName + " " + c.LastName);
  }
 }

// </snippetlinqexamples22>